mitmproxy/proxy/protocol/http.py

```diff
@@ -327,7 +330,15 @@ def _process_flow(self, f):
                 )
 
                 def get_response():
-                    self.send_request(f.request)
+                    if f.request.stream:
```

Comment:
Does `f.request.stream` get set to `True` anywhere? Or is this inherited from the flow?